src/sdd/MapHVAC.cpp

```diff
@@ -2719,7 +2719,12 @@ namespace sdd {
     auto htgSensEff75Element = element.child("HtgSensEff75");
     boost::optional<double> _htgSensEff75 = lexicalCastToDouble(htgSensEff75Element);
     if (_htgSensEff75) {
-      hx.setSensibleEffectivenessat75HeatingAirFlow(_htgSensEff75.get());
+      if (boost::optional<model::Curve> curve_ = hx.sensibleEffectivenessofHeatingAirFlowCurve()) {
+        if (boost::optional<model::TableLookup> sensibleEffectivenessofHeatingAirFlowCurve_ = curve_->optionalCast<TableLookup>()) {
+          double sensibleEffectivenessat100HeatingAirFlow = hx.sensibleEffectivenessat100HeatingAirFlow();
+          sensibleEffectivenessofHeatingAirFlowCurve_->setOutputValues({_htgSensEff75.get(), sensibleEffectivenessat100HeatingAirFlow});
+        }
+      }
```

Comment:
Ok, so it seems we do want to support **some** amount of backward compatibility in setting the 75% effectivness due to this MapHVAC stuff.

I don't think the current implementation here is good though. You're creating the object, so you're in control. You should create the curve instead of checking whether it exists.

Reply:
I think the opposite way (OSM > SDD) is missing too.